### DISCUSSION TITLE
Why does brave browsers dark mode invert the color of hint markers?

### DISCUSSION THREAD

**Discussion by txtyash** [gdh1995/vimium-c#509] [Q&A] [answered]
Brave browser has a built in option to select light or dark theme which inverts the color of the hint markers.

**Comment by gdh1995** [comment, marked as answer]
https://github.com/gdh1995/vimium-c/wiki/Style-the-UI-of-Vimium-C-using-Custom-CSS#work-with-enable-force-dark
